### DISCUSSION TITLE
Sending multiple plates to Klipper

### DISCUSSION THREAD

**Discussion by behold81** [OrcaSlicer/OrcaSlicer#446] [Ideas]
Is this something that can be designed to send multiple plates to Klipper in one go. if I have a project of 10 plates I can get them all sent to the printer ready to print one at a time. I am not sure if you can have it print them all too with a pause between plate exchange. might need a Klipper improvement.

**Comment by Crazor** [comment]
That would be insanely useful for large recurring prints!

**Comment by czyz** [comment]
Submitted a PR that adds a feature that could be used for this.
[Add Plate Changer Support to OrcaSlicer](https://github.com/OrcaSlicer/OrcaSlicer/pull/13177)
